File: langgraph/data/equiv/162.py
import hashlib

def convert_text_to_hash(input_string):
    if not input_string:
        return None
    encoded_string = input_string.encode('ascii')
    md5_hash = hashlib.md5(encoded_string).hexdigest()
    return md5_hash

# method name: string_to_md5 -> convert_text_to_hash
# parameter name: text -> input_string
# moved the import statement outside the function
# replaced the conditional expression with an if statement for clarity
# split the encoding and hashing steps for better readability

File: langgraph/result/qwen/14b-eq2js/162.js
const crypto = require('crypto');

function convertTextToHash(inputString) {
    if (!inputString) {
        return null;
    }
    const encodedString = Buffer.from(inputString, 'ascii');
    const md5Hash = crypto.createHash('md5').update(encodedString).digest('hex');
    return md5Hash;
}

// method name: string_to_md5 -> convertTextToHash
// parameter name: text -> inputString
// moved the import statement outside the function
// replaced the conditional expression with an if statement for clarity
// split the encoding and hashing steps for better readability